File: TIENDARENEGUL/src/Alimento.java
import java.util.Date;
class Alimento extends Producto {
    // Atributos y métodos específicos de Alimento
    private String fechaCaducidad;
    private int calorias;

    // Constructor
    public Alimento(String nombre, double precio, String fechaImportacion, String numeroSerie, String fechaCaducidadint, int calorias, int stock) {
        super(nombre, precio, fechaImportacion, numeroSerie, stock);
        this.fechaCaducidad = fechaCaducidad;
        this.calorias = calorias;
    }

    // Getter y Setter específicos
    public String getFechaCaducidad() {
        return fechaCaducidad;
    }

    public void setFechaCaducidad(String fechaCaducidad) {
        this.fechaCaducidad = fechaCaducidad;
    }

    public int getCalorias() {
        return calorias;
    }

    public void setCalorias(int calorias) {
        this.calorias = calorias;
    }
    /* Método específico para Alimento
    public void informacionNutricional() {
        // Implementación para mostrar información nutricional del alimento
        System.out.println("Información nutricional de " + getNombre() + ": " + getCalorias() + " calorías");
    }*/
}

File: TIENDARENEGULPY/Alimento.py
from Alimento import Producto  # Suponiendo que Producto es una clase existente en otro archivo llamado producto.py

class Alimento(Producto):
    def __init__(self, nombre, precio, fecha_importacion, numero_serie, fecha_caducidad, calorias, stock):
        super().__init__(nombre, precio, fecha_importacion, numero_serie, stock)
        self.fecha_caducidad = fecha_caducidad
        self.calorias = calorias

    def get_fecha_caducidad(self):
        return self.fecha_caducidad

    def set_fecha_caducidad(self, fecha_caducidad):
        self.fecha_caducidad = fecha_caducidad

    def get_calorias(self):
        return self.calorias

    def set_calorias(self, calorias):
        self.calorias = calorias

    # Método específico para Alimento
    def informacion_nutricional(self):
        # Implementación para mostrar información nutricional del alimento
        print("Información nutricional de", self.nombre + ":", self.calorias, "calorías")
